K/KronLinInv/Compat.toml

```diff
@@ -0,0 +1,3 @@
+[0]
+Documenter = "0.22"
```

Comment:
You probably don't need Documenter as a dependency.

Reply:
You're right, will fix that, thanks.